packages/webapp/src/utils/constants.js

```diff
@@ -52,6 +52,7 @@ const NO = 'no'
 const NO_AGAIN = 'noAgain'
 const OS_API_TOKEN = 'land/os-api-token'
 const PUBLIC = 'public'
+const PUBLIC_ROUTES = `${PUBLIC}-routes`
```

Comment:
Is it necessary to have a PUBLIC constant here? it's not used anywhere else now that PUBLIC_ROUTES is added to routes array.

Reply:
PUBLIC constant removed as suggested.